docs/upgrade-to-2.0.md

```diff
@@ -12,28 +12,28 @@ For a full, machine-generated compatibility report see the `docs/compat_reports`
     or Twitter SDKs. If you plan to enable either of these providers you should include the
     appropriate SDK in your app to avoid a crash. This saves space for developers only using
     Google/Email/Phone authentication.
-  * **Sign Out and Delete** - The `AuthUI#delete(Activity)` and `AuthUI#signOut(Activity)` 
+  * **Sign Out and Delete** - The `AuthUI#delete(Activity)` and `AuthUI#signOut(Activity)`
```

Comment:
I think there should be `out` as you set lowercase to `libraries` before

Reply:
Yeah, I actually haven't really cleaned up the upgrade guides, will do. (That was an automated find/replace)